javascript/javascript1/week3/Js1week3.js

```diff
@@ -0,0 +1,189 @@
+//Item array removal
+
+const names = [
+    "Peter",
+    "Ahmad",
+    "Yana",
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
+];
+const nameToRemove = "Ahmad";
+const indexNameToRemove = names.indexOf("Ahmad");
+names.splice(indexNameToRemove, 1);
+
+console.log(names); // ['Peter', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala'] */
+
+//When will we be there??
+
+const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+};
+
+function timeToDestination(speed, destinationDistance) {
+    arrivingTime = destinationDistance / speed;
+    diff = arrivingTime - Math.floor(arrivingTime);
+    if (diff == 0) {
+        return console.log(`You will arrive in ${arrivingTime} hours`);
+    }
+
+    else
+        return console.log(`You will arrive in ${Math.floor(arrivingTime)} hours and ${Math.round(diff * 60)} minutes`);
+}
+
+
+timeToDestination(50, 432); // 8 hours and 38 minutes
```

Comment:
Awesome 🔥
Please remember to declare you variables with either let and const on line 27 and 28. Would it make more sense to use let or const in this case? 

You are calling Math.floor(arrivingTime) twice, both on line 28 and line 34. To make the code even more readable, I would suggest you save the value of Math.floor(arrivingTime) in a variable, perhaps named const hours.
Additionally, you can consider whether your variable diff should in fact be named minutes. What makes the most sense to you? 😊